src/main/resources/assets/jei/lang/ru_RU.lang

```diff
@@ -54,16 +54,16 @@ config.jei.interface.recipeAnimationsEnabled.comment=Показывать ани
 
 config.jei.search=Поиск
 config.jei.search.comment=Настройки, отвечающие за поиск.
-config.jei.search.atPrefixRequiredForModName=Использовать @ для имени мода
-config.jei.search.atPrefixRequiredForModName.comment=Требовать "@ИмяМода" для поиска по имени модификации вместо просто "ИмяМода".
-config.jei.search.prefixRequiredForTooltipSearch=Использовать # для всплывающей подсказки
-config.jei.search.prefixRequiredForTooltipSearch.comment=Использовать "#" перед словом для поиска всплывающих подсказок.
-config.jei.search.prefixRequiredForOreDictSearch=Использовать $ для словаря руд
-config.jei.search.prefixRequiredForOreDictSearch.comment=Использовать "$" перед словом для поиска имён словаря руд.
-config.jei.search.prefixRequiredForCreativeTabSearch=Использовать % для имени творческой вкладки
-config.jei.search.prefixRequiredForCreativeTabSearch.comment=Использовать "%" перед словом для поиска имён творческих вкладок.
-config.jei.search.prefixRequiredForColorSearch=Использовать ^ для цветов
-config.jei.search.prefixRequiredForColorSearch.comment=Использовать "^" перед словом для поиска предмета по цвету.
+config.jei.search.modNameSearchMode=@ИмяМода
+config.jei.search.modNameSearchMode.comment=Режим поиска по имени мода (префикс: @)
+config.jei.search.tooltipSearchMode=#ВсплывающаяПодсказка
+config.jei.search.tooltipSearchMode.comment=Режим поиска по всплывающей подсказке (префикс: #)
+config.jei.search.oreDictSearchMode=$СловарьРуд
+config.jei.search.oreDictSearchMode.comment=Режим поиска по имени в словаре руд (префикс: $)
+config.jei.search.creativeTabSearchMode=%ТворческаяВкладка
```

Comment:
Yes please change this line to use `%%`
![2016-11-12_00 08 47](https://cloud.githubusercontent.com/assets/916092/20236512/4446f5c8-a86c-11e6-9902-c2e389c64e51.png)